pyproject.toml

```diff
@@ -784,7 +784,6 @@ select = [
     "S317", # suspicious-xml-sax-usage
     "S318", # suspicious-xml-mini-dom-usage
     "S319", # suspicious-xml-pull-dom-usage
-    "S320", # suspicious-xmle-tree-usage
```

Comment:
This rule has been deprecated and will be removed in the future

> This rule was deprecated as the lxml library has been modified to address known vulnerabilities and unsafe defaults. As such, the defusedxml library is no longer necessary, defusedxml has [deprecated](https://pypi.org/project/defusedxml/0.8.0rc2/#defusedxml-lxml) its lxml module.

From https://docs.astral.sh/ruff/rules/suspicious-xmle-tree-usage/